Formula/vg.rb

```diff
@@ -0,0 +1,50 @@
+class Vg < Formula
+  desc "Virtualgo: Easy and powerful workspace based development for go"
```

Comment:
Possibly just `desc "Easy and powerful workspace based development for go"`

Reply:
Please make this change. Also, I assume we want "Go" not "go"